examples/kubernetesversions/kubernetes1.5.3.json

```diff
@@ -3,7 +3,7 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorVersion": "1.5.3"

```

Comment:
Looks like 1.5.3 does not exist in ACS Engine anymore. If that's the case this file should be renamed to: kubernetes1.5.json